src/i18n/translations/english.json

```diff
@@ -0,0 +1,20 @@
+{
+  "navbar": {
+    "scholarships": "Scholarships",
+    "add": "Add",
+    "about": "About",
+    "contact": "Contact"
+  },
+
+  "btn": {
+    "browse": "Browse Scholarships",
+    "add": "Add a Scholarship",
```

Comment:
used in the other stacked PRs.
Included it here so I wouldn't have to add them for every PR.